lib/embeds.js

```diff
@@ -65,9 +65,16 @@ function renderInstagram (props) {
   </amp-instagram>);
 }
 
+const relativeSrcRegex = /^\/\//;
+
 function renderCustom (props) {
   const { src, width, height, secure } = props;
 
+  // AMP doesn't allow relative-url iframes,
+  // always embed them as https
+  const secureSrc = relativeSrcRegex.test(src)
```

Comment:
I'd do src.slice(0, 2) === '//' it'll be more performant than regex and easier to read I think.